src/openst_deployer.js

```diff
@@ -0,0 +1,77 @@
+const Web3 = require('web3');
+const Package = require('@openstfoundation/openst.js');
+
+const logger = require('./logger');
+
+class OpenSTDeployer {
```

Comment:
Why not just `OpenST` (and then `OpenST::deployXxx()`)

Reply:
Good point 👍 Renaming done!